### DISCUSSION TITLE
Possible to add worker nodes of different types?

### DISCUSSION THREAD

**Discussion by jeffreyflynt** [edgelesssys/constellation#2862] [Q&A] [answered]
I see under `worker_default` in the config we can specify `instanceType`; however, can we add different types to the workers? Or would it be best to just manually add an instance type to the scale set in Azure?

**Comment by thomasten** [comment, marked as answer]
Hi,
Yes, you can define multiple worker node groups. Please see https://docs.edgeless.systems/constellation/workflows/config#creating-additional-node-groups

**Reply by jeffreyflynt** [reply]
Thanks, totally missed that section.
